learning.yml

```diff
@@ -119,9 +115,9 @@
 
     - role: ome.omero_web_apps
       omero_web_apps_packages:
-        - omero-gallery==3.1.1
-        - omero-iviewer==0.8.1
-        - omero-virtual-microscope==1.0.1
+        - omero-gallery=={{ omero_web_apps_release["omero_gallery"] }}
```

Comment:
FYI if you have plain variable names you can also use `{{ omero_web_apps_release.omero_gallery }}`

Reply:
Leaving that to the follow up.

Reply:
:+1: